eq-author/src/App/QuestionnairesPage/QuestionnairesView/QuestionnairesTable/index.js

```diff
@@ -1,24 +1,28 @@
 import React, { useState } from "react";
-import styled from "styled-components";
 import PropTypes from "prop-types";
 import { propType } from "graphql-anywhere";
 import gql from "graphql-tag";
 
-import { SORT_ORDER } from "../constants";
+import { SORT_ORDER } from "constants/sort-order.js";
+import * as Headings from "constants/table-headings";
 
-import Row from "./Row";
-import TableHead from "./TableHead";
+import QuestionnaireTable from "components/QuestionnaireTable";
 import Panel from "components/Panel";
 
+import tableHeadings from "../../../../components/QuestionnaireTable/TableHeadings";
+
```

Comment:
You shouldn't need to pass these in as a prop to the component now, as they live alongside it. Instead, you can import them directly into the component file.